src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -1,5 +1,17 @@
-(function(global, doc, eZ, $) {
+(function (global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
+    const config = {
+        attributes: false,
+        childList: true,
+        subtree: true,
+    };
+    const observer = new MutationObserver((mutationsList, observer) => {
+        mutationsList.forEach((mutation) => {
+            if (mutation.removedNodes.length && doc.querySelector('.ez-tooltip.show')) {
```

Comment:
`doc.querySelector('.ez-tooltip.show')` could be put into variable, as it's used two times